mount/mount.go

```diff
@@ -21,12 +21,27 @@ type Mount struct {
 	Datastore datastore.Datastore
 }
 
+type MountSlice []Mount
```

Comment:
you *could* use the new `sort.Slice` api, which saves you a good bit of code and a type. Though either way is fine with me